utils/storage.py

```diff
@@ -9,8 +9,15 @@
 import scipy
 import json
 import os
-import glob
-import tarfile
+from rich import print
+
+
+def isfloat(x):
+    return isinstance(x, float)
+
```

Comment:
again don't like having single line functions. I also couldn't see anywhere in the code where this is called